packages/core/parcel-bundler/src/scope-hoisting/shake.js

```diff
@@ -85,7 +76,8 @@ function isExportAssignment(path) {
     // match "path.any = any;"
     path.parentPath.isMemberExpression() &&
     path.parentPath.parentPath.isAssignmentExpression() &&
-    path.parentPath.parentPath.node.left === path.parentPath.node
+    path.parentPath.parentPath.node.left === path.parentPath.node &&
+    path.parentPath.parentPath.get('right').isPure()
```

Comment:
Should rather be called `isExportAssignmentPure` now?

Reply:
actually... I think this check is unnecessary now